### PR TITLE
Fix config loading race condition

### PR DESCRIPTION
**Pull request by gwenneg**
This could be the solution to the prod config loading issue.

I was able to reproduce the issue locally. It only happens when the app is started with a `java -jar` command.

With the changes from this PR, I was no longer able to reproduce it.

### CODE REVIEW

File: external/src/main/java/com/redhat/cloud/policies/engine/AlertStarter.java [hunk 1]
```diff
@@ -64,56 +58,21 @@ void startApp(@Observes StartupEvent startup) {
...
-                .whenComplete((empty, error) -> {
-                    if(error != null) {
-                        LOGGER.error("Engine start failed", error);
-                        alerts.stop();
-                    } else {
-                        pluginRegister.init();
-                        ((StatusServiceImpl) statusService).setStarted(true);
-                        LOGGER.info("Started Policies Engine");
-                    }
-        }).join();
```
Comment (gwenneg) [inline external/src/main/java/com/redhat/cloud/policies/engine/AlertStarter.java:117]: This was nothing more than a sequential execution using a thread from `ForkJoinPool`.

File: external/src/main/java/org/hawkular/alerts/AlertsStandalone.java [hunk 2]
```diff
@@ -153,36 +153,34 @@ public AlertsStandalone() {
...
-
-            partitionManager.init();
-            alertsContext.init();
-            dataDrivenGroupCacheManager.init();
-            actionsCacheManager.init();
-            publishCacheManager.init();
-            extensions.init();
-            engine.initServices();
-        });
+    public void init() {
```
Comment (gwenneg) [inline external/src/main/java/org/hawkular/alerts/AlertsStandalone.java:156]: This method is only called by `AlertStarter#initialize` which doesn't seem async execution. It's ok to drop the `CompletableFuture<Void>` return type.

**Review by gwenneg — COMMENTED** [review]
Review with `Hide whitespace changes` checked for an easier review.

**Comment by pilhuhn** [comment]
I can't comment if this fixes the code or not, but I certainly like the change, as it simplifies the code.

**Comment by josejulio** [comment]
> I can't comment if this fixes the code or not, but I certainly like the change, as it simplifies the code.

ditto

**Review by josejulio — APPROVED** [review]
lgtm

**Pull request merged**